lib/zendesk_apps_support/package.rb

```diff
@@ -121,6 +121,31 @@ def translation_files
       files.select { |f| f =~ %r{^translations/} }
     end
 
+    def obj(options)
+      begin
+        app_id = options.fetch(:app_id)
+        asset_url_prefix = options.fetch(:assets_dir)
+        name = options.fetch(:app_name)
+      rescue KeyError => e
+        raise ArgumentError, e.message
+      end
+
+      app_class_name = "app-#{app_id}"
+
+      {
+        app_class_name: app_class_name,
+        app_class_properties: manifest.app_class_properties,
+        asset_url_prefix: asset_url_prefix,
+        author: manifest.author,
+        framework_version: manifest.framework_version,
+        id: app_id,
+        location_icons: location_icons,
+        logo_asset_hash: generate_logo_hash(manifest.products),
```

Comment:
I think there might have been a pending action to decide whether or not this could be handled by ZAF.  In particular, I believe that this line should be replaced by manifest.products and this logic https://github.com/zendesk/zendesk_apps_support/pull/266/files#diff-f9dc5da4f07a7cd59a608b6da769f0ecR295-R302 moved to a yet-to-be-written-function `ZendeskApps.initInstalledJs` in ZAF.